tests/chart_tests/test_probes.py

```diff
@@ -4,6 +4,143 @@
 from tests import supported_k8s_versions
 
 ignore_kind_list = []
+ignore_list = [
```

Comment:
If we don't want to maintain this big ignore list we can probably just `@pytest.mark.skip` this whole test for now.

Reply:
Or maybe skip specific components, not specific tests?